dynawo/sources/API/CRT/test/CMakeLists.txt

```diff
@@ -21,12 +21,6 @@ target_link_libraries(${MODULE_NAME}
                       dynawo_API_CRT
                       dynawo_Test)
 
-if(CXX11_ENABLED)
```

Comment:
I think this is required in centos7

Reply:
The linked library dynawo_Test uses interface dynawo_Test_utils which is linked to LibXml2::LibXml2.
This is transitive.